modules/customizing-parameters-serving-runtime.adoc

```diff
@@ -0,0 +1,35 @@
+:_module-type: PROCEDURE
+
+[id="customizing-parameters-serving-runtime{context}"]
+= Customizing the parameters of a deployed model-serving runtime
+
+[role='_abstract']
+You can modify the parameters of an existing runtime to suit your business requirements.
+
+NOTE: Customizing the parameters of a runtime only affects the model deployment selected.
+
+.Prerequisites
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges.
+* You have deployed a model-serving runtime on the single model-serving platform.
```

Comment:
I think this will be `You have deployed a model on the single model-serving platform.` Users can only see models in the Models tab, Model serving runtimes can only be viewed as an administrator in the Settings in the left navigation. Users cannot deploy a model without a model-serving runtime(it's a required field) in the Kserve `Deploy` modal.